explorer/client/package.json

```diff
@@ -24,9 +31,9 @@
   },
   "main": "app.js",
   "proxy": "http://localhost:8080",
-  "dependencies": {},
-  "devDependencies": {
-    "@babel/preset-typescript": "^7.3.3",
+  "dependencies": {
+    "@babel/core": "7.4.3",
+    "@babel/preset-typescript": "~7.3.3",
```

Comment:
Why are we restricting this to minor only?

Reply:
It was conflicting with the definition in the root `package.json` https://github.com/smartcontractkit/chainlink/blob/develop/package.json#L30

Reply:
Lets use yarn `resolution` for this?